extensions/default/src/customizations/loadingIndicatorProgressCustomization.tsx

```diff
@@ -0,0 +1,5 @@
+import { LoadingIndicatorProgress } from '@ohif/ui';
+
+export default {
+  'ui.LoadingIndicatorProgress': LoadingIndicatorProgress,
+};
```

Comment:
can you follow our naming convension which is something.somethingElseElse for all of these